static/js/apps/explore/app.tsx

```diff
@@ -220,7 +220,7 @@ export function App(props: { isDemo: boolean }): JSX.Element {
       isPendingRedirect = shouldSkipPlaceOverview(pageMetadata);
       if (isPendingRedirect) {
         const placeDcid = pageMetadata.place.dcid;
-        const url = `/place/${placeDcid}`;
+        const url = `/dev-place/${placeDcid}`;
```

Comment:
undo this until we are ready to launch. in the meantime, we could add another redirect in flask that does this based on environment

Reply:
Good catch- reverted